docs/content/architectuur/zgw-in-gegevenslandschap.md

```diff
@@ -0,0 +1,188 @@
+
+Enkele voorbeelden:
+1. Een gemeente wil alle Meldingen Openbare Ruimte op een kaart tonen. Vraag je hiertoe alle melding-zaken op met een generiek ZDS-koppelvlak? Wetende dat de AVG dit voor het overgrote deel aan zaken/zaaktypen niet toestaat? Of kies je voor een meldingen-specifiek koppelvlak, dat mogelijk functioneel rijker is: type melding, prioriteit van de melding, tekstuele toelichting…allemaal wellicht nuttig om op een dergelijke kaart te kunnen weergeven. Maar niet of slechts met moeite (zaaktypespecifieke eigenschappen) met een generiek zaken-koppelvlak te realiseren.
+2. Hoe wil je een proceseigenaar/afdelingshoofd van management-informatie voorzien? Op basis van zaken kun je erg generieke metrieken weergeven. Aantal zaken per afdeling, aantal zaken per status, zaken die voor of achter lopen op plandatum etc. Toch is het vergelijken van een subsidiezaak met een vergunnningenzaak appels met peren vergelijken. En zit een manager vergunningen vermoedelijk op heel andere metrieken te wachten: aantal vergunningen per wijk, aantal afhandeluren per bouwvolume-eenheid etc.
+3. Hoe wil je status-informatie weergeven aan de klant? Heel generiek obv enkel voorgedefinieerde “zaakstatussen”, of wil je dit functioneel rijker doen in een domeinspecifiek portal, bijvoorbeeld specifiek voor subsidies?
+
+Het is belangrijk om hier als gemeente even goed bij stil te staan. In veel gevallen zal het nog steeds handig blijken om iets met zaken op te lossen, of misschien beide: generieke informatie via zaken tonen en als aanvulling daarop enkele functioneel rijkere portalen of apps per domein.
+
+### 1.4.8. Notificeren over zaken
+
+In een gegevenslandschap willen we geen zaken (of andere informatie) meer “rondsturen”. Vraag is wel hoe een behandelaar met een taakapplicatie te weten komt dat er een melding voor hem beschikbaar is die behandeld moet worden. Er zijn nog andere scenario’s te bedenken, waarin het nuttig is dat een partij actief wordt genotificeerd over een zaak(status). Bijvoorbeeld bij meerdere behandelaars die met meerdere systemen werken (of moeten we dat niet willen?), of wanneer er een nieuw document aan een zaak is toegevoegd door een KCC-medewerker. Het is nuttig om enkele van deze scenario’s uit te werken binnen de context van een gegevenslandschap en hiervoor een generieke oplossing voor te stellen. Een verdere uitwerking van https://www.gemmaonline.nl/index.php/ZGW_in_GEMMA_2_compleet#Notificeren_over_Zaken binnen een gegevenslandschap.
```

Comment:
Als developer kan ik nog niet zo veel met het stukje achter de link op GEMMA. Het is niet concreet genoeg om iets te maken wat zo maar werkt voor alle componenten. Hier moet gekozen worden voor een bepaalde techniek, wat je volgens mij ook voorstelt?

Dus, wanneer trekken we [PubSub](https://en.wikipedia.org/wiki/Publish%E2%80%93subscribe_pattern) binnen scope :)

Reply:
Een paar user stories over notificatie zouden enorm helpen. Als medewerker wil ik een signalering krijgen als er een nieuwe melding groenvoorziening is aangemaakt zodat ... Dat vertaalt zich dan in een notificatie van de ZAC door ZRC of door de MRC (MeldingRegistratieComponent). Zoiets?

Reply:
aah, die link over PubSub :-)
Dat notificeren in GEMMA 2 moesten we inderdaad nog verder uitwerken..moeten we daar echt een techniek voor kiezen?
ik vind het een goed idee.. @Hugo-ter-Doest voer jij die US op?